oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -293,4 +299,35 @@ private static Options options() {
 
         return options;
     }
+
+    private static boolean checkForMultipleOptions(CommandLine cmd){
+        int optionsCount = 0;
+        if (cmd.hasOption("l")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("oxd_id")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("d")) {
+            optionsCount ++;
+        }
+        if(optionsCount > 1)
+            return true;
+        else
+            return false;
+    }
+
+    private static boolean checkForArgsAfterListParameter(String[] args){
+
+        String[] parameterList = {"-d", "-oxd_id", "-a"};
```

Comment:
Shouldn't we include `-c` parameter? Probably it's safe construct it from `org.gluu.oxd.server.Cli#options` instead of declaring it again.

Reply:
Options constructed from org.gluu.oxd.server.Cli#options.